packages/react-reconciler/src/ReactFiberClassComponent.js

```diff
@@ -193,6 +194,9 @@ const classComponentUpdater = {
       update.callback = callback;
     }
 
+    if (revertPassiveEffectsChange) {
```

Comment:
did you forget to import this flag in this module? failing tests for the same reason.

Reply:
Oops, yes